arch/arm/boot/dts/overlays/overlay_map.dts

```diff
@@ -187,6 +187,10 @@
 		bcm2711;
 	};
```

Comment:
You should insert `bcm2712 = "pcie-32bit-dma-pi5";` here to make it load the Pi 5 variant automatically.

Reply:
Resolved, thanks for the tip.